<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioMainWgtClass</class>
 <widget class="QWidget" name="AudioMainWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioMainWgt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="tab_AudiofileOpen">
      <attribute name="title">
       <string>文件导入</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_AudioFileOpenTab"/>
     </widget>
     <widget class="QWidget" name="tab_recorder">
      <attribute name="title">
       <string>录音</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="HLayout_tab_recorder">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>691</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_FFTTest">
      <attribute name="title">
       <string>FFT</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_FFT_Test"/>
     </widget>
     <widget class="QWidget" name="tab_STFT">
      <attribute name="title">
       <string>STFT</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_STFT"/>
     </widget>
     <widget class="QWidget" name="tab_EffOscMask">
      <attribute name="title">
       <string>有效震荡屏蔽</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_EffOscWgt"/>
     </widget>
     <widget class="QWidget" name="tab_FFTWnd">
      <attribute name="title">
       <string>窗函数分析</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_FFT_Wnd"/>
     </widget>
     <widget class="QWidget" name="tab_SNRSelect">
      <attribute name="title">
       <string>BDSNR</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_SNRSelWgt"/>
     </widget>
     <widget class="QWidget" name="tab_EffSigBuffer">
      <attribute name="title">
       <string>有效信号缓冲区</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_EffSigBuffWgt"/>
     </widget>
     <widget class="QWidget" name="tab_FDR">
      <attribute name="title">
       <string>倍频相关性</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_FDRWgt"/>
     </widget>
     <widget class="QWidget" name="tab_BandWidthInput">
      <attribute name="title">
       <string>宽带输入</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_BandWidthInput"/>
     </widget>
     <widget class="QWidget" name="tab_AudioObject">
      <attribute name="title">
       <string>听觉对象判定</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_AudioObject"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
